logger/interface.go

```diff
@@ -0,0 +1,26 @@
+package logger
+
+import "io"
+
+// Logger defines the behavior of a data logger component
+type Logger interface {
+	Trace(message string, args ...interface{})
```

Comment:
Do we need a loglevel None? So that nothing is logged?

Reply:
Done